kubernetes.yaml

```diff
@@ -0,0 +1,153 @@
+package:
+  name: kubernetes
+  version: 1.27.3
+  epoch: 2
```

Comment:
start this at `2` so it doesn't squash the existing `kubectl-x-r1` we already have